apps/dav/lib/Connector/Sabre/ExceptionLoggerPlugin.php

```
@@ -92,6 +92,12 @@ public function logException(\Exception $ex) {
 			//app or so.
 			return null;
 		}
+		//If the exception is InsufficientStorage, then log a debug message and return null
+		if (($ex->getPrevious() === null) && ($ex instanceof Exception\InsufficientStorage)) {
```

Why this? `$this->nonFatalExceptions` contains nonFatal exceptions which will be logged in debug mode

---

@patrickjahns  Oops, I guess I took the wrong path. I was trying to suppress the exception message too. My bad. I have updated the code.

---

Might have misunderstood me there - maybe your approach was the right one - was trying to understand why not using already existing functionality

---

@patrickjahns In this https://github.com/owncloud/core/issues/31284#issue-317988198 , `This should be probably treated like a normal log message`, was the reason I thought to suppress the message. But in fact you are right, considering this https://github.com/owncloud/core/issues/31284#issuecomment-399905245. Treating the exception message as debug one. The current change does what is required ;)